lib/generators/repack/install_generator.rb

```diff
@@ -33,7 +34,11 @@ def copy_package_json
     end
     def copy_webpack_conf
       copy_file "webpack.config.js", "config/webpack.config.js"
-      copy_file "webpack.config.heroku.js", "config/webpack.config.heroku.js"
+      puts 'Are you going to be deploying to heroku? (yes \ no)'
+      if gets.strip.downcase == 'yes'
```

Comment:
I think this should be if gets.strip.downcase =~ /y(es)?/
so we can cover if someone just types y